app/services/course_search.rb

```diff
@@ -0,0 +1,50 @@
+class CourseSearch
```

Comment:
Our other services use the suffix `Service`, can we bring this one in line? Can be in another PR.